tests/src/test/scala/whisk/core/entity/test/ExecHelpers.scala

```diff
@@ -33,7 +33,7 @@ trait ExecHelpers
     self: Suite =>
 
     private val config = new WhiskConfig(ExecManifest.requiredProperties)
-    ExecManifest.initialize(config) shouldBe true
+    ExecManifest.initialize(config).isSuccess shouldBe true
```

Comment:
Should rewrite to: `ExecManifest.initialize(config) should be a 'success` to print a non-conforming value on failure (and for fancyness 😁 )